src/MojiraBot.ts

```diff
@@ -41,20 +43,29 @@ export default class MojiraBot {
 			// Register events.
 			EventRegistry.setClient( this.client );
 			EventRegistry.add( new ErrorEventHandler() );
-			EventRegistry.add( new MessageEventHandler( this.client.user.id ) );
+			EventRegistry.add( new RemoveReactionEventHandler( this.client.user.id ) );
 
 			const rolesChannel = this.client.channels.get( BotConfig.rolesChannel );
 			if ( rolesChannel && rolesChannel instanceof TextChannel ) {
 				try {
 					await rolesChannel.fetchMessage( BotConfig.rolesMessage );
-					for ( const channel of BotConfig.requestChannels ) {
-						const requestChannel = this.client.channels.get( channel );
-						if ( requestChannel && requestChannel instanceof TextChannel ) {
-							await requestChannel.fetchPinnedMessages();
+
+					const internalChannels = new Map<string, TextChannel>();
```

Comment:
All of this only happens if there's a roles channel. We should probably move it outside of the if block, but since I'm reworking the config stuff anyway, it's not that important (we have a roles channel set after all)